### PR TITLE
Cockpit 207

### PR DESCRIPTION
**Pull request by KKoukiou**
Preview https://kkoukiou.github.io/cockpit-project.github.io/blog/cockpit-207.html

### CODE REVIEW

File: _posts/2019-11-13-cockpit-207.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+slug: cockpit-207
+category: release
+summary: Cockpit with authentication and storage page improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.
+
+### Cockpit's web server now accepts EC certificates
```
Comment (martinpitt) [inline _posts/2019-11-13-cockpit-207.md]: Let's drop the "now" from the headings: "Web server: Accept EC certificates"

File: _posts/2019-11-13-cockpit-207.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+To specify the [Elliptic Curve Cryptography](https://wiki.openssl.org/index.php/Elliptic_Curve_Cryptography) certificate
+the web service should use, simply drop a file with the extension .cert in the `/etc/cockpit/ws-certs.d` directory.
+
+### Storage: All Software Devices will be listed now in a single panel
```
Comment (martinpitt) [inline _posts/2019-11-13-cockpit-207.md]: "List all software devices in a single panel"

Comment (garrett) [reply _posts/2019-11-13-cockpit-207.md]: I guess with the prefix, like:

"Storage: List all software devices in a single panel"

...right?

Comment (martinpitt) [reply _posts/2019-11-13-cockpit-207.md]: Perhaps add a rationale with a condensed variant of the https://github.com/cockpit-project/cockpit/pull/11809 description? Maybe "This avoids showing empty panels for types of devices that you don't have".

**Comment by garrett** [comment]
@martinpitt's suggested headings look fine to me.

File: _posts/2019-11-13-cockpit-207.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.
+
+### Cockpit's web server now accepts EC certificates
+
+To specify the [Elliptic Curve Cryptography](https://wiki.openssl.org/index.php/Elliptic_Curve_Cryptography) certificate
+the web service should use, simply drop a file with the extension .cert in the `/etc/cockpit/ws-certs.d` directory.
```
Comment (martinpitt) [inline _posts/2019-11-13-cockpit-207.md]: That's a little ambiguous -- *all* certificates need to end in .cert. Maybe

The /etc/cockpit/ws-certs.d` directory can now contain [ECC] certificates in addition to the classic RSA ones.

File: _posts/2019-11-13-cockpit-207.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+### Storage: All Software Devices will be listed now in a single panel
+
+![Storage software devices](/images/storage-software-devices.png){:.screenshot}
+
+### Redesigned notifications
+
+This release of Cockpit features a new notification design.
+The new design corresponds to the [alert component from PatternFly 4](https://pf4.patternfly.org/components/Alert/examples/).
```
Comment (martinpitt) [inline _posts/2019-11-13-cockpit-207.md]: To avoid repetition: "new notification design, which corresponds to .."?

File: _posts/2019-11-13-cockpit-207.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+![Alert firewall](/images/alert-firewall.png){:.screenshot}
+![Alert ipv4 settings](/images/alert-ipv4-settings.png){:.screenshot}
+![Alert diagnostic report](/images/alert-diagnostic-report.png){:.screenshot}
```
Comment (martinpitt) [inline _posts/2019-11-13-cockpit-207.md]: I know you are proud of this (and you should be), but maybe one or two are enough :grin:

File: _posts/2019-11-13-cockpit-207.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+### Try it out
+
+Cockpit 207 is available now:
+
+ * [For your Linux system](https://cockpit-project.org/running.html)
+ * [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/207)
+ * [Cockpit Fedora 30](https://bodhi.fedoraproject.org/updates/FEDORA-2019-a460236814)
+ * [Cockpit Fedora 31](https://bodhi.fedoraproject.org/updates/FEDORA-2019-0667dd1786)
```
Comment (martinpitt) [inline _posts/2019-11-13-cockpit-207.md]: I recommend XXXXing the URLs for now, as we don't have them yet. Makes it more obvious that they still need to be changed.

File: _posts/2019-11-13-cockpit-207.md (hunk 3 above)
Comment (garrett) [inline _posts/2019-11-13-cockpit-207.md]: OK, this is kind of klunky due to the sentence construction. It's not wrong, but it's hard to parse.

Drop has multiple meanings. The cert has ECC, but that's happenstance for putting a cert in that directory, right?

What about something like this?

```suggestion
Use a certificate with [Elliptic Curve Cryptography](https://wiki.openssl.org/index.php/Elliptic_Curve_Cryptography) by copying the certificate file into the `/etc/cockpit/ws-certs.d` directory. Make sure the certificate file has the `.cert` extension.
```

Please feel free to correct or riff off what I typed above. :wink:

(This suggestion won't merge cleanly, as the paragraph is split into two lines instead of relying on editor wrapping. This is one of the reasons we want to use auto-wrap for Markdown files in the repo.)

File: _posts/2019-11-13-cockpit-207.md (hunk 5 above)
Comment (KKoukiou) [reply _posts/2019-11-13-cockpit-207.md]: Hehe sorry

File: _posts/2019-11-13-cockpit-207.md (hunk 3 above)
Comment (KKoukiou) [reply _posts/2019-11-13-cockpit-207.md]: @martinpitt  has posted another suggestion, I will use his since he wrote the feature better than all since he wrote that.

File: _posts/2019-11-13-cockpit-207.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+summary: Cockpit with authentication and storage page improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.
+
+### Web server: Accept EC certificates
+
+The /etc/cockpit/ws-certs.d` directory can now contain
```
Comment (martinpitt) [inline _posts/2019-11-13-cockpit-207.md]: Missing the initial ` here
